lib/schlepp/format/base.rb

```diff
@@ -66,9 +86,36 @@ def process!
         @after.call if @after
       end
 
+      def glob_to_regexp(glob)
+        escaped = Regexp.escape(glob)
+          .sub('\*\*', '.*')
+          .sub('\*', '(?:(?!\\/).)*')
```

Comment:
Looks like this is only supporting `*` and `**`? We've got other glob patterns like this one:
https://github.com/ElasticSuite/spyder-spice/blob/04ed000f0fd3883c86e40ee1071428aff459d5fe/import/inventory_import.rb#L93-L97

I wonder if there's a general glob 2 regex library we can use here.

Reply:
I guess we only need to support what `Dir.glob` supports: https://ruby-doc.org/core-2.1.0/Dir.html#method-c-glob

This might work, looks like it supports everything but `[]` (but that works the same as regex so it's probably ok): https://github.com/alexch/rerun/blob/master/lib/rerun/glob.rb

Reply:
Ah, good call. Thought we could get away with just `*` and `**`. I saw a promising glob to regex written in node.

Doesn't the one you linked get `*` and `**` mixed up? I thought `*` shouldn't match subdirectories infinitely, but that one's replacing it with `.*`. If it did, we wouldn't be able to move images to `imported`without them being picked up anyway.

Reply:
Yeah I don't necessarily trust that guy's code. It's likely that it's full of bugs.